content/api/commands/clearlocalstorage.md

```diff
@@ -46,8 +46,7 @@ Pass in an options object to change the default behavior of
 
 ### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
 
-<List><li>`cy.clearLocalStorage()` yields
-`null`.</li><li>`cy.clearLocalStorage()` cannot be chained further.</li></List>
+- `cy.clearLocalStorage()` yields `null`.
```

Comment:
Is end a query or a command?

Reply:
It's a command.

It yields null though, I don't think it needs to be called out explicitly that you can chain things off of it.